src/components/molecules/Teaser/Teaser.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { FC, ReactNode } from 'react'
+import { Link } from '../../utility/Link'
+
+import styles from './teaser.module.css'
+
+export type TeaserProps = {
+  label: string
+  heading: string
+  text: string
+  linkUrl: string
+  linkText?: string
+  image: ReactNode
+}
+
+export const Teaser: FC<TeaserProps> = ({
+  label,
+  heading,
+  text,
+  linkUrl,
+  linkText,
+  image,
+}) => {
+  const randomInt = Math.floor(Math.random() * 1000)
```

Comment:
This is fine here, but just as future React ref using `Math.random` isn't the greatest for keys. It will cause an entire rerender of the component rather than updating its instance. Check out the docs: https://reactjs.org/docs/reconciliation.html#trade-offs

If Gatsby is giving you some kind of unique ID in GraphQL, that would be ideal.

Reply:
Thanks! I refactored this to use a unique ID (provided by Gatsby) in the next PR. I did an actual map in that PR, so I needed a key anyway. So I used it as the key, and also as the ID for aria stuff here. 👍